gradle.properties

```diff
@@ -4,17 +4,17 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 # check these on https://fabricmc.net/develop/
-minecraft_version=1.21.1
-yarn_mappings=1.21.1+build.3
-loader_version=0.15.11
+minecraft_version=1.21.3
+yarn_mappings=1.21.3+build.2
+loader_version=0.16.9
 
 # Mod Properties
-mod_version=1.7.2-1.21.1
+mod_version=2.0.0-1.21.3
 maven_group=com.github.crimsondawn45
 archives_base_name=FabricShieldLib
 
 # Dependencies
-fabric_version=0.102.1+1.21.1
-mod_menu_version=11.0.1
+fabric_version=0.106.1+1.21.3
+mod_menu_version=12.0.0-beta.1
 midnightlib_version=1.5.8-fabric
```

Comment:
newer versions came out:

fabric_version=0.112.1+1.21.3
midnightlib_version=1.6.4-fabric

Reply:
Let's keep the dependency version of this mod as it is and should only pump up the version when we absolutely have to. Using the latest version of Fabric API may prevent some mod packs on Curseforge from using our FabricShieldLib.

Reply:
The versions I mentioned are about 2 weeks and 2 months old respectively, so I think it would be more compatibility-safe if we updated them.